test/test_pallas.py

```diff
@@ -477,6 +505,147 @@ def test_flash_attention_backward(self):
     for i in [(q, q_grad), (k, k_grad), (v, v_grad)]:
       self.assertTrue(mse(i[0].grad.cpu(), i[1].cpu()) < 1e-4)
 
+  @unittest.skipIf(xr.device_type() != 'TPU' or tpu.version() < 4,
+                   "This test only works on TPUv4+.")
+  def test_paged_attention_wrapper(self):
+    jax.config.update('jax_default_matmul_precision', jax.lax.Precision.HIGHEST)
```

Comment:
It's interesting that you use jax as the reference. I guess that works too. Wondering if we can just the eager attention helper in the class instead? Or that doesn't work? Anyway, if you are using jax as the reference, you can drop this.

Reply:
Sg, yeah I saw that we're dependent on JAX Pallas anyways, so I thought it may be easier to just test against the JAX's outputs. 

Ah, makes sense removed the `jax.config` updates.